packages/clerk-js/src/ui/elements/Action/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { ActionCard } from './ActionCard';
+import { ActionClosed } from './ActionClosed';
+import { ActionOpen } from './ActionOpen';
+import { ActionRoot } from './ActionRoot';
+import { ActionTrigger } from './ActionTrigger';
+
+const Action = {
+  Root: ActionRoot,
+  Card: ActionCard,
+  Trigger: ActionTrigger,
+  Open: ActionOpen,
+  Closed: ActionClosed,
+};
+
+export default Action;
```

Comment:
@desiprisg let's avoid the default export here. I'd suggest just a named `Action` export, and then import it like:
```
import { Action } from '../../elements/Action';
```